docs/swagger.yaml

```diff
@@ -941,7 +946,8 @@ components:
           description: 'The URL of the checkout frontend. Example: "https://vippscheckout.vipps.no/v1/".'
         pollingUrl:
           type: string
-          description: 'The URL to poll for session information. Example: "https://api.vipps.no/checkout/v1/session/31gf1g413121".'
+          description: "The URL to poll for session information. Example: "https://api.vipps.no/checkout/v1/session/31gf1g413121"."
+      additionalProperties: { }
```

Comment:
Are there other additionalProperties? if not, maybe remove it

Reply:
Removing it, I was already working on cleaning up after the merging of the autogenerated swagger.